update-rebrandly-link/action.yml

```diff
@@ -0,0 +1,56 @@
+name: "Update Rebrandly Link"
+inputs:
+  apikey:
+    description: 'API Key to access Rebrandly'
+    required: true
+  domain:
+    description: 'Domain name used for link'
+    required: true
+  slashtag:
+    description: 'Slashtag value for link'
+    required: true
+  destination:
+    description: 'New destination for link'
+    required: true
```

Comment:
Do we have documentation somewhere on where to find these values or what needs to be provided (and maybe when this action needs to be run?)

Reply:
I added a README file.  Let me know if I have to clarify anything.